tmux/demos/medical/Kitware-M2.yml

```diff
@@ -20,7 +20,7 @@ root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
 # on_project_start: command
 on_project_start: |
   export ROS_NAMESPACE=${ROS_NAMESPACE:-/kitware}
-  export HL2_IP=${HL2_IP:-192.168.1.4}
+  export HL2_IP=${HL2_IP:-192.168.4.2}
```

Comment:
Note that everyone ends up setting this IP when we are running things locally, so this variable often gets thrashed. This *is* reading from an optional preexisting environment variable, so there is a world where the system specific IP is something that is configured on the system instead of this being modified all the time. This is not currently set up to happen, but I know how it could be.

Reply:
For now, I think fixing this to match the standard demo setup is a good idea - let's revisit in the future.